packages/scripts/package.json

```diff
@@ -90,6 +90,10 @@
 		"webpack-cli": "^4.9.1",
 		"webpack-dev-server": "^4.4.0"
 	},
+	"peerDependencies": {
```

Comment:
This is a bit concerning, but I think it happens because of Enzyme.

Reply:
Maybe we can list them as optional peer deps, so that it's not needed if the users don't run the lint command?

Reply:
Here's exactly why it was showing up:

```sh
$ yarn explain peer-requirements p1e560
➤ YN0000: @wordpress/scripts@npm:22.0.1 doesn't provide react, breaking the following requirements:

➤ YN0000: @wojtekmaj/enzyme-adapter-react-17@npm:0.6.6 [536d8] → ^17.0.0-0          ✘
➤ YN0000: @wojtekmaj/enzyme-adapter-utils@npm:0.1.4 [74ca6]    → ^17.0.0-0          ✘
➤ YN0000: @wordpress/jest-preset-default@npm:8.0.1 [41561]     → ^17.0.0            ✘
➤ YN0000: react-shallow-renderer@npm:16.14.1 [5f24f]           → ^16.0.0 || ^17.0.0 ✘
➤ YN0000: react-test-renderer@npm:17.0.2 [74ca6]               → 17.0.2             ✘

➤ YN0000: Note: these requirements start with @wordpress/jest-preset-default@npm:8.0.1 [41561]
```

`@wordpress/jest-preset-default` already had peer deps, probably from the enzyme adapter, like you said:

https://github.com/WordPress/gutenberg/blob/f833f821105aa7129533fff802937d22f14c3099/packages/jest-preset-default/package.json#L40-L44

Reply:
Ok, it makes sense. Thank you for a follow-up @noahtallen.